### ISSUE TITLE
Case studies/use case

### ISSUE THREAD

**Issue by gclawson1** [Sustainable-Aquafeeds-Project/feed_biodiv_impact_mapping#20]
Today we chatted a bit about the best way to demonstrate how our results could be used. Currently, I have an example in the manuscript which takes a look at the cell with the maximum impact, and dives into the composition of species present and raw materials contributing to it:

 > For the first example, we identify areas with large biodiversity impacts. Our analysis reveals that the highest average impact in the world occurs within a cell in Norway, with an average proportion of habitat area impacted of 0.22 for the species present, under the plant-dominant scenario. This cell encompasses 39 impacted species out of 97 species with habitat presence, none of which are categorised as threatened by the IUCN. While the impact magnitude in this cell is considerable (~a quarter of habitat impacted), the absence of threatened species suggests that the habitat displacement is unlikely to have substantial consequences on species prevalence overall. This underscores the importance of comprehensive understanding regarding species impacts, particularly concerning their conservation status. Had the impacted species within this cell been categorised as threatened, it would be important for feed companies to consider shifting their sourcing to less biodiverse or threatened locations, if possible. We acknowledge that this information alone may not deter production in highly biodiverse areas, as farmers may opt to sell to alternative buyers or industry. However, the potential loss of a customer could incentivise farmers to invest in sustainable practices or adjust crop locations. Most importantly, our findings could serve as evidence for feed companies striving to meet biodiversity standards, such as the Corporate Sustainable Reporting Directive (CSRD). This directive has standards, the European Sustainability Reporting Standards (ESRS), which have a focus on biodiversity and ecosystems. The biodiversity standard, ESRS E4, mandates that by 2030 all types of companies in Europe will have to measure and report their impact on biodiversity. Our results are a first step in providing this type of information vital to these biodiversity objectives.

We decided that perhaps an even better exploration would be to pick out 3 or 4 larger hotspots of impact and look at the composition of the impacts. 

This is a map showing impacts in Norway that are >=95th quantile of impacts. The colors represent the raw material that contributes most to the impact in that cell, and the alpha is the average proportion of habitat impacted (our impact metric shown in the global map).

I'm still trying to determine the best way to visualize/describe the composition of threatened (or not) species in these examples, but for now I've just calculated the percentage of total impacts shown which are on a threatened species (e.g., not least concern or data deficient). 

![image](https://github.com/Sustainable-Aquafeeds-Project/feed_biodiv_impact_mapping/assets/33332753/55122705-0a09-49cb-9e1c-61a90dd9110e)

Basically what we're aiming for is an expansion of what I currently have in the manuscript, with a couple more examples of hotspots (Canada, Chile, Argentina, Norway).
